### DISCUSSION TITLE
With NextJS 13

### DISCUSSION THREAD

**Discussion by IRediTOTO** [nextauthjs/next-auth#5648] [Help]
Hi, NextJS 13 just come out. How can we add `next-auth` to it? Someone do this?

**Comment by ymc9** [comment]
create-next-app uses Next 13 now

**Comment by raulmarindev** [comment]
There is an issue related to this: https://github.com/nextauthjs/next-auth/issues/5650

**Comment by Tarbez** [comment]
I'm trying to setup nextAuth in my new nextjs13 project but things are different somehow and there's very little documentation or video on how to achieve this. I have opened a an issue [here](https://github.com/vercel/next.js/discussions/42134)

Does anyone have an example code that I could look at ?

**Reply by agustif** [reply]
This repo uses next auth w nextjs 13 https://github.com/shadcn/taxonomy/blob/main/pages/api/auth/%5B...nextauth%5D.ts
